sample/Kmd.Logic.Cpr.Client.Sample/Program.cs

```diff
@@ -96,6 +102,45 @@ private static async Task Run(AppConfiguration configuration)
                 var citizen = await cprClient.GetCitizenByCprAsync(configuration.CprNumber).ConfigureAwait(false);
 
                 Log.Information("Citizen data: {@Citizen}", citizen);
+
+                var isSubscribeByCprSuccess = await cprClient.SubscribeByCprAsync(configuration.CprNumber).ConfigureAwait(false);
```

Comment:
isSubscribeByCprSuccess is quite long winded. Can't we just use `success` and then reuse the variable?

Reply:
Changed as suggested